internal/policies/mount/mount_test.go

```diff
@@ -183,9 +255,26 @@ func TestApplyPolicy(t *testing.T) {
 					e.Key = tc.keys[i]
 					secondEntries = append(secondEntries, e)
 				}
+				m.SetSystemCtlCmd(mockSystemCtlCmd(t, tc.secondSystemCtlFailingArgs...))
```

Comment:
I don't think there is any difference beyond efficiency. But I would rather keep it that way, since it's the same behavior as the one used in the manager_test.go for the general manager.